SU-SWS/stanford_paragraph_types
# HSDO-1069

# READY FOR REVIEW

# Summary
- Insets text for Teaser Cards at an earlier breakpoint

# Needed By (Date)
- End of sprint (10/11)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that the text in Teaser Cards insets beginning at 1199px
3. This branch can be seen in action at https://sites.stanford.edu/handacenter (see homepage Teaser Cards)

# Affected Projects or Products
- H&S
- stanford_paragraph_types
- stanford_story_page

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/HSDO-1069

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
